### ISSUE TITLE
Cant config LEDnetWF in Home Assistant

### ISSUE THREAD

**Issue by Goaheadz** [raulgbcr/lednetwf_ble#2]
Probably missing something stupid, but I cannot had the device to HomeAssistant.

After installing this into HAOS:
Device & Services-> Add integration->I Select "LEDnetWF BLE."
I insert bluetooth MAC and give it the name.
I get "! unkown error" when I press "submit"

I can scan the device and find it with my phone, but bluetooth HA cant.
Using available bluetooth from RPI4.
Any special code needed to be added to configuration.yaml?

Guidance needed

Thx
Goahead

**Comment by raulgbcr** [comment]
Hi, could you check your logs for any other messages?
Also what light are you using, this has only been tested on a couple of AliExpres lamps, so may not be fully compatible with yours.

Regards

**Comment by raulgbcr** [comment]
Latest update should maybe fix your issue, please check.

**Comment by Goaheadz** [comment]
Hi @raulgbcr 

Without this last update, I was able to find the device in HA ( RPI4+HAOS).
I needed to power on bluetooth via terminal. Could not find any information or instructions, but I manage to enable everything, and the BLE RGB controler was found...solved.
The issue after that was the "no entities" like reported in another threat.

So, After updating LEDnetWF with your last version, entity was created, and I can now power on/off, dimmer and "change color".

![image](https://github.com/raulgbcr/lednetwf_ble/assets/13796554/1b3f9e70-5c6e-42a1-a7f8-e153fb41a399)

Still have to check color changes. I may have some bad connections in strip, since I can only light green.
But it should be an HW problem.

Thx
G.
